libafl_qemu/src/drcov.rs

```diff
@@ -0,0 +1,202 @@
+use std::{path::PathBuf, sync::Mutex};
+
+use hashbrown::{hash_map::Entry, HashMap};
+use libafl::{inputs::UsesInput, state::HasMetadata};
+use libafl_targets::drcov::{DrCovBasicBlock, DrCovWriter};
+use rangemap::RangeMap;
+use serde::{Deserialize, Serialize};
+
+use crate::{
+    blocks::pc2basicblock,
+    emu::GuestAddr,
+    helper::{QemuHelper, QemuHelperTuple, QemuInstrumentationFilter},
+    hooks::QemuHooks,
+    Emulator,
+};
+
+static mut DRCOV_IDS: Mutex<Option<Vec<u64>>> = Mutex::new(None);
```

Comment:
If you leave out the `mut` here, you can remove the `unsafe` blocks.
Since mutexes use interior mutability, there is no need for mut here.

Reply:
Cc @TeumessianFox